packages/data-api/src/__tests__/testFetchDataElements.test.ts

```diff
@@ -6,14 +6,9 @@ import { getTestDatabase } from '@tupaia/database';
 
 import { TupaiaDataApi } from '../TupaiaDataApi';
 
-export const testFetchDataElements = () => {
+describe('fetchDataElements()', () => {
   const api = new TupaiaDataApi(getTestDatabase());
 
-  it('throws an error with invalid parameters', async () => {
-    await expect(api.fetchDataElements()).toBeRejectedWith(/data element codes/);
-    await expect(api.fetchDataElements(null)).toBeRejectedWith(/data element codes/);
-  });
```

Comment:
No longer need these kinds of tests, thanks to TS doing type checking 😄

Reply:
Wouldn't this be validation on the api interface itself? Rather than inside-the-package type checking? `fetchDataElements()` is a public method.

Validation code which throws the error:
https://github.com/beyondessential/tupaia/pull/3776/files#diff-afd345b68ba1a10d317d519770e12d7674c23a54f14a37f90945c6c263b7e013R77

Reply:
That's a good point! I think if our entire codebase were typescript, then I'd argue it'd be worth removing that validation code and just making the type signature a bit stricter. But since this gets called from `@tupaia/data-broker` which is still js, I'll re-add the tests back 👍

Reply:
Makes sense, more motivation to go full TS :)